packages/react/src/components/DataTable/TableExpandHeader.js

```diff
@@ -49,6 +49,21 @@ const TableExpandHeader = ({
   );
 };
 
+function isRequiredIfPropExists(requiredProp) {
+  function checker(props, propName, componentName) {
+    if (props[requiredProp] === undefined) {
```

Comment:
Thank you for jumping in @tw15egan - Tried to add `console.log('requiredProp:', requiredProp); console.log('propName:', propName)`, where `requiredProp` and `propName` seem to be always the same. So wondering what `isRequiredIfPropExists()` is meant for... Any more details? Thanks!

Reply:
Had a typo, should be checking if `enableExpando` exists, then those props are required